File: Vepr-bot/models/selectedproductcharacteristics.py
class Selectedproductcharacteristics:

	def __init__(self):
		self.id = 0
		self.productcharacteristics = 0
		self.selectedproduct = 0

	@property
	def _id(self):
		return self.id

	@_id.setter
	def _id(self, value):
		self.id = value



	@property
	def _productcharacteristics(self):
		return self.productcharacteristics

	@_productcharacteristics.setter
	def _productcharacteristics(self, value):
		self.productcharacteristics = value



	@property
	def _selectedproduct(self):
		return self.selectedproduct

	@_selectedproduct.setter
	def _selectedproduct(self, value):
		self.selectedproduct = value


	def get(self):
		selectedproductcharacteristicsDict = self.__dict__
		del selectedproductcharacteristicsDict['id']
		selectedproductcharacteristicsDict['idProductcharacteristics'] = selectedproductcharacteristicsDict.pop('productcharacteristics')
		selectedproductcharacteristicsDict['idSelectedproduct'] = selectedproductcharacteristicsDict.pop('selectedproduct')
		return selectedproductcharacteristicsDict

	def set(self, id, productcharacteristics, selectedproduct):
		self.id = id
		self.productcharacteristics = productcharacteristics
		self.selectedproduct = selectedproduct
		return self

File: vepr_backend/lib/model/selected_product_characteristics.dart
import 'package:conduit/conduit.dart';
import 'package:vepr_backend/model/product_charateristics.dart';
import 'package:vepr_backend/model/selected_product.dart';


class SelectedProductCharacteristics extends ManagedObject<_Selected_Product_Characteristics> implements _Selected_Product_Characteristics {}
class _Selected_Product_Characteristics {
  @primaryKey
  int? id;

  @Serialize(input: true, output: false)
  int? idSelectedProduct;

  @Serialize(input: true, output: false)
  int? idProductCharacteristics;

  @Relate(#selectedProductCharacteristicsList, isRequired: true, onDelete: DeleteRule.cascade)
  SelectedProduct? selectedProduct;

  @Relate(#selectedProductCharacteristicsList, isRequired: true, onDelete: DeleteRule.cascade)
  ProductCharacteristics? productCharacteristics;
}